src/ui/screens/settings/ViewNetworkSettings.qml

```diff
@@ -65,23 +65,6 @@ VPNViewBase {
             }
         }
 
-        VPNCheckBoxRow {
-            id: tunnelPort53
-            objectName: "settingTunnelPort53"
-            width: parent.width - VPNTheme.theme.windowMargin
-            showDivider: true
-
-            labelText: VPNl18n.SettingsTunnelPort53
-            subLabelText: VPNl18n.SettingsTunnelPort53Description
-            isChecked: (VPNSettings.tunnelPort53)
-            isEnabled: vpnFlickable.vpnIsOff
-            onClicked: {
-                if (vpnFlickable.vpnIsOff) {
-                    VPNSettings.tunnelPort53 = !VPNSettings.tunnelPort53
```

Comment:
Do we want to remove the setting flag too?

Reply:
yep! https://github.com/mozilla-mobile/mozilla-vpn-client/pull/4946